src/main/java/net/sf/jabref/bst/VM.java

```diff
@@ -144,11 +160,6 @@ public void execute(BstEntry context) {
                     throw new VMException("Can only compare two integers with >");
                 }
 
```

Comment:
Both o1 and o2 are Integers (otherwise an Exception would have been thrown on line 160), so should be safe to only compare them using `compareTo`.